examples/custom_distribution.ipynb

```diff
@@ -0,0 +1,425 @@
+{
```

Comment:
<p>And tests?</p><p> </p><p>And maybe to consider contributing it with a PR ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>